src/transport_performance/gtfs/gtfs_utils.py

```diff
@@ -48,17 +58,20 @@ def bbox_filter_gtfs(
     _is_expected_filetype(
         pth=out_pth, param_nm="out_pth", check_existing=False
```

Comment:
This would benefit from having type defences for out_pth and in_pth. Current behaviour does not reinforce the type hinting.
Suggested changes are using:  
`_type_defence(pth, "pth", (pathlib.Path, str))`